lib/receipt.js

```diff
@@ -25,23 +25,32 @@ const _recipientNetFees = new WeakMap();
 const _netTransfer = new WeakMap();
 const _singleTransfer = new WeakMap();
 
-const PAYMENT_RECEIPT_HASHED_PROPERTIES = [
-    'seals.wallet.signature.v',
-    'seals.wallet.signature.r',
-    'seals.wallet.signature.s',
-    'nonce',
-    'sender.nonce',
-    'sender.balances.current',
-    'sender.balances.previous',
-    'sender.fees.single.currency',
-    'sender.fees.single.amount',
-    'sender.fees.net.*.currency',
-    'recipient.nonce',
-    'recipient.balances.current',
-    'recipient.balances.previous',
-    'recipient.fees.net.*.currency',
-    'transfers.single',
-    'transfers.net',
+const PAYMENT_RECEIPT_HASH_PARTS = [
+    [
+        'seals.wallet.signature.v',
+        'seals.wallet.signature.r',
+        'seals.wallet.signature.s'
+    ],
+    [
+        'nonce'
+    ],
+    [
+        'sender.nonce',
+        'sender.balances.current',
+        'sender.balances.previous',
+        'sender.fees.single.amount',
+        'sender.fees.single.currency.ct',
+        'sender.fees.single.currency.id'
+    ],
```

Comment:
The `sender.fees.net` is no longer needed?

Reply:
The hashing of sender.fees.net is currently commented out in the smart contracts:

```
// TODO Consider adding dynamic size 'payment.sender.fees.net' to exchange hash
// payment.sender.fees.net
// TODO Consider adding dynamic size 'payment.recipient.fees.net' to exchange hash
// payment.recipient.fees.net
```

Reply:
https://github.com/hubiinetwork/striim-contracts/blob/6e5ace06e81c5dbdb8fc5f06be5fc7600823adef/test/mocks.js#L452